pom.xml

```diff
@@ -66,6 +69,31 @@
 			<version>${vertx-unit.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>io.vertx</groupId>
+			<artifactId>vertx-kafka-client</artifactId>
+			<version>${vertx-kafka-client.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
```

Comment:
I would prefer remove it for now if not necessary in order to have a consistent patch. We'll add the Kafka client then, with the other related PR.